### DISCUSSION TITLE
Is there `linspace`?

### DISCUSSION THREAD

**Discussion by med1844** [tracel-ai/burn#2578] [Q&A] [answered]
Same as title, something similar to `torch.linspace` would be great.

I tried to search in issues, discussion, book, doc.rs and didn't find anything similar to this.

For now, I'm using this temporary solution:

```rust
    fn linspace(v_min: f32, v_max: f32, steps: usize, device: &B::Device) -> Tensor<B, 1> {
        let arange = Tensor::<B, 1, Int>::arange(0..steps as i64, device);
        arange.float() * (v_max - v_min) / (steps - 1) as f64 + v_min
    }
```

Edit: fix `linspace` bug

**Comment by laggui** [comment, marked as answer]
There is no `linspace` equivalent atm, so your solution is a good way to achieve that :)
